collections/_helpers/colours.html

```diff
@@ -127,6 +150,46 @@ <h5 class="margin--bottom--lg has-text-primary">Background colours</h5>
 <p class="has-background-{{ colour.name }}">
     Use the <code>has-background-{{ colour.name }}</code> colour class to set background colour
 </p>
+{%- endhighlight -%} 
+                </td>
+            </tr>
+        {%- endfor -%}
+    </tbody>
+</table>
+
+<hr class="margin--bottom--lg margin--top--lg" />
+
+<h5 class="margin--bottom--lg has-text-primary has-text-weight-bold">Component colours</h5>
+<p>Use the <code>is-*color*</code> class to set the component colors for:</p>
+
+<ul>
+    <li><a href="/doc/accordion/">Accordion</a></li>
```

Comment:
href attributes here should point to /docs/...., not /doc/...

Reply:
this is fixed f733bdc